streams/src/main/java/org/apache/kafka/streams/processor/internals/TopologyMetadata.java

```diff
@@ -224,6 +225,7 @@ public void maybeWaitForNonEmptyTopology(final Supplier<StreamThread.State> thre
         } finally {
             unlock();
         }
+        log.info("Removed NamedTopology {} and updated topology version to {}", topologyName, version.topologyVersion.get());
```

Comment:
line 220 has the same log

Reply:
There's a semantic difference but it's hard to notice, the first one says "Remov_ing_" when we start removing it, whereas this one says "Remov_ed_" to indicate that we have finished. That said it's probably worth clarifying further, eg `"Finished removing..."` -- I'll update it to make the purpose of each line more clear